war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -157,16 +148,38 @@ public void dropOrganizationMetadata(final String organizationId, final String m
     *       id of the new organization to create
     * @param organizationName
     *       name of the new organization to create
+    * @throws OrganizationAlreadyExistsException
+    *       when organization with organizationId already exists
     */
-   public void createOrganization(final String organizationId, final String organizationName) {
+   public void createOrganization(final String organizationId, final String organizationName) throws OrganizationAlreadyExistsException {
+      if (dataStorage.collectionHasDocument(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(organizationId))) {
+         throw new OrganizationAlreadyExistsException(ErrorMessageBuilder.organizationAlreadyExistsString(organizationId));
+      }
       DataDocument document = new DataDocument(LumeerConst.Organization.ATTR_ORG_ID, organizationId)
             .append(LumeerConst.Organization.ATTR_ORG_NAME, organizationName)
-            .append(LumeerConst.Organization.ATTR_ORG_DATA, new DataDocument())
-            .append(LumeerConst.Organization.ATTR_USERS, new DataDocument());
+            .append(LumeerConst.Organization.ATTR_ORG_DATA, new DataDocument());
       dataStorage.createDocument(LumeerConst.Organization.COLLECTION_NAME, document);
    }
 
    /**
+    * Changes organization id
+    *
+    * @param oldOrganizationId
+    *       id of the organization to change
+    * @param newOrganizationId
+    *       new id for organization
+    * @throws OrganizationAlreadyExistsException
+    *       when organization with newOrganizationId already exists
+    */
+   public void updateOrganizationId(final String oldOrganizationId, final String newOrganizationId) throws OrganizationAlreadyExistsException {
+      if (dataStorage.collectionHasDocument(LumeerConst.Organization.COLLECTION_NAME, organizationIdFilter(newOrganizationId))) {
+         throw new OrganizationAlreadyExistsException(ErrorMessageBuilder.organizationAlreadyExistsString(organizationId));
+      }
+      DataDocument document = new DataDocument(LumeerConst.Organization.COLLECTION_NAME, newOrganizationId);
+      dataStorage.updateDocument(LumeerConst.Organization.COLLECTION_NAME, document, organizationIdFilter(oldOrganizationId));
```

Comment:
Isn't it somehow possible to detect if the organization with the same ID exists during `updateDocument` operation? I mean checking some specific MongoDB exception so we do not have to make two database calls.